types/figgy-pudding.d.ts

```diff
@@ -0,0 +1,75 @@
+declare module "figgy-pudding" {
```

Comment:
this whole module is a heroic effort of types and I can't even read most of it. Amazing!

Reply:
This is @forivall's great effort. I merely forked here idea and tweaked it a bit. 😍 🔥

Reply:
There's a PR up in the `figgy-pudding` repo, so I guess see what happens with that. In the interim, including the types for that package here is fine.

Reply:
Sadly, I no longer own `figgy-pudding` myself (it belongs to $VCPM), and I need to lock down that particular repo, which is only there for archival purposes.

Reply:
In that case, a PR to DefinitelyTyped makes sense. And as mentioned you can test the types, https://github.com/zkat/figgy-pudding/pull/13#issuecomment-509454099

Reply:
Ive submitted lots of typedefs to definitely typed, and I could take my old pr, update it and submit it there - especially as I've improved in my typedefs writing skills, and I'm sure they'd improvements to make